src/app/legacy/components/RadioSchedule/ScheduleItemHeader/index.jsx

```diff
@@ -151,7 +151,7 @@ ScheduleItemHeader.propTypes = {
   brandTitle: string.isRequired,
   link: string.isRequired,
   state: string.isRequired,
-  startTime: number.isRequired,
+  startTime: oneOfType([number, string]).isRequired,
```

Comment:
Since we have made changes to legacy should there be a refactor ticket for this now to bring it to new component standards and create typescript types for an upcoming sprint?

Reply:
We have decided not to upgrade the radio schedule at this time / ever, as we believe it's like the Most Read component, and we're not willing to spend the time on bringing it up to standards. We believe that the component design might completely change in the future, and if it does, we would do the upgrade then.

Reply:
Okay cool maybe the standards need upgraded for this scenario then? I feel like this isn't the last time it'll happen. We have pretty complex components still to work on in the future. If a new RS is on the roadmap then that makes sense. Thanks for clarifying